### ISSUE TITLE
[Enhancement Feature] Wireless Adapter Support

### ISSUE THREAD

**Issue by Pik-a-boo** [shonumi/gbe-plus#102]
Adding support for Wireless Adapter and have it work in conjunction with the netplay feature.
The only other emulator who can do that is VBA-Link (and maybe VBA-M) but its archaic and old.

**Comment by shonumi** [comment]
Yes, I definitely have this on my TODO list of hardware to support. It's very well documented, thankfully. First, however, I'd have to get regular multiplayer on the GBA (via the Link Cable) running, but once that's sorted I'll take a look at the Wireless Adapter.

**Comment by KostaSaizo7** [comment]
I am wondering if this is still in the TODO list. Recently, pizza boy pro for android achieved stable connection. 

Maybe this research can be of help: https://blog.kuiper.dev/gba-wireless-adapter
